pages/products/[id].js

```diff
@@ -1,20 +1,51 @@
+import ShareLink from "../../components/ShareLink/ShareLink";
 import supabase from "../../utils/supabaseClient";
+import {useState, useEffect} from "react";
+import React from "react";
+import { withCoalescedInvoke } from "next/dist/lib/coalesced-function";
```

Comment:
withCoalescedInvoke is used in this products/[id].js page?

Reply:
good call! what does this do exactly?

Reply:
Thanks for pointing this out! It shouldn't be in there, it doesn't do anything it's just an error I forgot to delete. I've removed it now and everything still works fine.